### ISSUE TITLE
[Feature Request] output_keep_open option

### ISSUE THREAD

**Issue by igorlfs** [benlubas/molten-nvim#19]
I'm using the `molten_output_virt_lines` option and I think I'd be nice if we could sort of "always show" the output cells, using the virtual lines, similar to an actual Jupyter Notebook. It'd allow, for instance, comparisons between different cell outputs. I'm not really aware of an alternative within the neovim ecosystem.

**Comment by benlubas** [comment]
This is an idea that I've had as well. I think it would be pretty neat. Unfortunately it's a decent chunk of work and I haven't really had the time to implement it as I wouldn't use it myself.

If you wanted to take a shot at it I'd be happy to point you in the right direction, otherwise I'll keep this open and if others would like this feature as well they can leave :+1: on it and I'll get to it when I do.

**Comment by benlubas** [comment]
Okay, I've spent a few days thinking about this and playing around with a few things, and I've settled on not implementing this as keeping multiple floating windows open, but instead by adding an option that will show the output as virtual text. (I know this is confusing, but the current `output_virt_lines` option just adds blank virtual lines below the output window. it doesn't display the output as virtual text).

The biggest advantages in this approach will be: 
- performance, I've noticed that moving floating windows around a lot can slow down the editor, and I don't think that's something I want to amplify by keeping many output windows open at once
- ease of implementation, the current code makes the assumption that there is only ever one open output window and the cursor has to be above it (ie. the output can't go off the top of the screen

With the small disadvantage of formatting probably not looking consistent between the virtual text and the output window.
